src/processor/mapper/map_extend.cpp

```diff
@@ -211,11 +213,12 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapLogicalRecursiveExtendToPhysica
     if (extend->trackPath()) {
         auto pathVectorPos = DataPos(outSchema->getExpressionPos(*rel));
         dataInfo = std::make_unique<RecursiveJoinDataInfo>(outDataPoses, colIndicesToScan,
-            inNodeIDVectorPos, outNodeIDVectorPos, tmpDstNodePos, true /* trackPath */,
-            pathVectorPos);
+            inNodeIDVectorPos, outNodeIDVectorPos, lengthVectorPos, tmpDstNodePos,
+            true /* trackPath */, pathVectorPos);
```

Comment:
Ditto (from my previous PR comments) to replace bools with enums to increase readability.